### DISCUSSION TITLE
How do you ignore vhosts with low number of hits?

### DISCUSSION THREAD

**Discussion by Andargor** [allinurl/goaccess#2486] [Q&A] [answered]
I'm sorry if I missed it in the docs, but I have several vhosts, and sometimes in the logs instead of my vhosts there are random IP addresses (maybe spoofing) with low hit counts. This makes the virtual hosts graph look like this, with all my vhosts squished on the left side:
![XDUUH3y](https://user-images.githubusercontent.com/3580755/224561989-3ee8b3dd-10ec-4e0b-b810-ee24b19e68f4.png)

Is it possible to ignore low counts? Whitelist my vhosts? Only show the top 95% of hits?

Any other solution?

**Comment by allinurl** [comment, marked as answer]
If you are getting IPs instead of vhosts, then it sounds like you either have the wrong format or a mix of log formats. Are you sure those IPs are not being logged instead of the vhost? 

As far as ignoring data, please take a look at the man page for some examples, e.g.,

    # grep -v "`cat exclude_vhost_list_file`" vhost_access.log | goaccess - --log-format=COMBINED -o report.html

**Comment by Andargor** [comment]
Logs are all from an nginx proxy, so it's the same format. It seems it's some crawler called "Go-http-client" that is spoofing it's source address, example of a proper vhost and one of the ones with an unknown IP in the vhost field:

```
www.andargor.com 64.124.8.50 - - [21/Feb/2023:13:49:48 +0000] "GET /kotor/kotor42-2_pre.jpg HTTP/1.1" 200 2820 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"
5.157.60.85 5.75.200.140 - - [21/Feb/2023:13:49:50 +0000] "POST / HTTP/2.0" 503 197 "-" "Go-http-client/2.0"
```

Thanks for the tip for filtering logs, I thought that it could be done within goaccess, but grep will do. Marking answered.

**Reply by allinurl** [reply]
Got it. As far as filtering, please stay tune for **#117** (working on it as we speak).
